packages/grafana-ui/src/components/Icon/cached.json

```diff
@@ -1,6 +1,8 @@
 [
   "unicons/at",
   "unicons/adjust-circle",
+  "unicons/align-left",
```

Comment:
unless this icons are directly required on a first page load it is better not to add them to this list as it'll increase the js bundle without a good reason.

Reply:
>They don't load at all if they are not there. I mean if you use grafana/ui outside of grafana you just have to bundle them.

True. I thought this was going to be used only inside the grafana context.